extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/HueBridge.java

```diff
@@ -862,7 +867,7 @@ private void requireAuthentication() {
     }
 
     // Used as assert in all requests to elegantly catch common errors
-    private void handleErrors(Result result) throws IOException, ApiException {
+    public void HandleErrors(Result result) throws IOException, ApiException {
```

Comment:
Why do you change the first letter of the method name to upper case?

Reply:
I always write pulic methods with a capital Letter, but I can change this if you have other guidelines.

Reply:
I would claim that we follow the "normal" Java naming converntion.
http://www.oracle.com/technetwork/java/codeconventions-135099.html

> Methods should be verbs, in mixed case with the first letter lowercase,  with the first letter of each internal word capitalized.